fuse/build.gradle.kts

```diff
@@ -8,16 +8,14 @@ dependencies {
     val diskLruCacheVersion: String by project
     val resultVersion: String by project
     val junitVersion: String by project
-    val hamcrestVersion: String by project
     val jsonVersion: String by project
 
     implementation(kotlin("stdlib", kotlinVersion))
```

Comment:
[s] You can remove it :) - https://kotlinlang.org/docs/reference/whatsnew14.html#dependency-on-the-standard-library-added-by-default

Reply:
Fixed! :)